allure-pytest/src/plugin.py

```diff
@@ -161,11 +175,13 @@ def pytest_configure(config):
 
 
 def select_by_labels(items, config):
+    config.option.allure_labels = set().union(*config.option.allure_labels)
```

Comment:
Is this conversion actually needed? I think we can leave the config option as is here. The conversion to a set just below should be enough. Or did I miss something?
```suggestion
```

Reply:
Actually now `config.option.allure_labels` is list of sets containing tuples (right after args collection).

line 178 is converting it to set of tuples because:
1. Set is unhashable type, so we cannot union list of sets (that's why I use unpacking here)
2. I wanted to make `config.option.allure_labels` variable contain labels using same format as `config.option.allure_features`, `config.option.allure_stories`, etc. in case it will be used somewhere else.

I think I can change `config.option.allure_labels` to make it list of tuples (by changing `cf_type` method).
It will allow to use it without unpacking. 
I know that `arg_labels` var will contain set of tuples.
However I am not sure if it is ok to leave `config.option.allure_labels` itself as list of tuples.

Let me know what you think. Do we need `config.option.allure_labels` to be same format as other labels for some other purpose (excluding `select_by_labels` method)? Or we can leave it as it comes after `action="append"` args collection.

Reply:
UPD: `extend` is not available for python 3.7, so I stick with `append` and list of lists containing tuples and *unpacking

Reply:
Yea, I see, `extend` is not the option. Your current solution is good IMO. Let's stay the config option intact for now.